avogadro/vtk/CMakeLists.txt

```diff
@@ -11,26 +11,24 @@ endif()
 
 find_package(Qt5 COMPONENTS Widgets REQUIRED)
 
-find_package(VTK
-  COMPONENTS
-    vtkRenderingOpenGL2 vtkGUISupportQt vtkDomainsChemistry
-    vtkRenderingVolumeOpenGL2 vtkViewsCore vtkRenderingFreeType
-  REQUIRED)
+find_package(VTK REQUIRED)
```

Comment:
We should stick to requesting the relevant modules, not just all of VTK - this will lead to overlinking, and making it less clear when modules we need are not built.